test/orgasm/compiler_vm_test.cpp

```
@@ -1269,6 +1415,236 @@ TEST_CASE("vm should dispatch imported symbols to registered native fallback", "
+          "[OrgasmTest][ModuleArtifact][Ngo][TypeCheck]")
+{
+  SourceModuleFixture fixture;
+
+  BytecodeModule staleBytecode;
+  staleBytecode.name = "pkg.math";
+  staleBytecode.exports["answer"] = 0;
+  staleBytecode.exportTypeReprs["answer"] = "fun () -> i32";
+  Function staleAnswer;
+  staleAnswer.name = "answer";
+  staleAnswer.code = {static_cast<uint8_t>(OpCode::PUSH_I32), 1, 0, 0, 0, static_cast<uint8_t>(OpCode::RETURN)};
+  staleBytecode.functions.push_back(std::move(staleAnswer));
```

Cppcheck (2.20.0)

[error] Uninitialized variables

(uninitvar)